src/actions/google/ads/lib/Customer_Match_Test_File_-_Contact_Information.csv

```diff
@@ -0,0 +1,52 @@
+# Instructions:
```

Comment:
This file is provided by Google as an example of the data normalization & hash procedure. I added it to the code to help serve as documentation. I also hope to use it directly within the automated test suite to validate our logic.